aws/resource_aws_instance_test.go

```diff
@@ -601,8 +601,8 @@ func TestAccAWSInstance_noAMIEphemeralDevices(t *testing.T) {
 			}
 
 			// Check if the root block device exists.
-			if _, ok := blockDevices["/dev/sda1"]; !ok {
-				return fmt.Errorf("block device doesn't exist: /dev/sda1")
+			if _, ok := blockDevices["/dev/xvda"]; !ok {
+				return fmt.Errorf("block device doesn't exist: /dev/xvda")
```

Comment:
is it guaranteed the root device will always be `/dev/xvda` ? since the root device for `hvm` virtualization types can be one or the other (sda1 or xvda) depending on the AMI, should we account for both? or the `testAccLatestAmazonLinuxHvmEbsAmiConfig` data-source generated always guarantees the block device?

Reply:
I've never seen Amazon Linux use sda1 but that doesn't mean it's impossible. I feel pretty safe with this though.